generate_data/netcdf/src/main/java/zarr_implementations/netcdf/App.java

```diff
@@ -0,0 +1,118 @@
+package zarr_implementations.netcdf;
+
+import com.bc.zarr.*;
```

Comment:
`com.bc.zarr` is still jzarr. The classes you are looking for will be in [ucar.nc2.iosp.zarr](https://github.com/Unidata/netcdf-java/blob/799f4ab239f4ea70da0c2a230c28f5594e4fe54c/cdm/zarr/src/main/java/ucar/nc2/iosp/zarr/ZArray.java)